tests/Unit/Provider/PhoneNumberProviderTest.php

```diff
@@ -54,6 +54,14 @@
 //                'phoneNumber' => '+0164742671',
```

Comment:
please dont comment tests- why they are commented  ? If not needed remove them and replace with new ones

Reply:
They were commented to with commit "disabled unit tests till we find a way to make it work on github"
Adding those cases to be available with fix

Reply:
Whole test is commented due to not working on github

Reply:
arrg, its not working due to it does not find PRestaShop global entity classes, or any clases from PRestaShop.

You can try adding this in unit test

```php

if (!class_exists(\Validate)) { 
  class Validate { 
    // copy methods from Presta that you use in test
   
  }

 }

```

Reply:
i do not have tests configured for this mollie :(